torchrl/objectives/dqn.py

```diff
@@ -164,7 +179,7 @@ class _AcceptedKeys:
 
     default_keys = _AcceptedKeys()
     default_value_estimator = ValueEstimators.TD0
-    out_keys = ["loss"]
+    out_keys = ["loss_objective"]
```

Comment:
this is bc-breaking
Maybe we could loosed the check above to `key.startswith("loss_") or key == "loss"` and make this change in another PR in a smoother manner?

Reply:
reverted